src/auxiliary/auxiliary.jl

```diff
@@ -243,7 +243,7 @@ macro threaded(expr)
     #     Look at the comments for `wrap_array` when considering to change this macro.
 
     return esc(quote
-                   Trixi.@batch $(expr)
+                   Polyester.@batch $(expr)
```

Comment:
ExplicitImports.jl complained about `@batch` being unused from Polyester.jl because it was prefixed with `Trixi.`. Was there any reason for that?

Reply:
I think it may have something to do with me not spending enough time to figure out macro hygiene. I wanted to enable users of Trixi.jl to use `@threaded` in their own code outside of Trixi.jl where Polyester is not necessarily in the namespace.

Reply:
Please check whether that works with the new version

Reply:
Ah, I see. This failing test also seems to be related, isn't it: https://github.com/trixi-framework/Trixi.jl/actions/runs/8295956873/job/22704074590?pr=1875#step:7:4053?

Reply:
Yes, exactly 👍

Reply:
Yes, as you said:
current version of PR:
```julia
julia> using Trixi

julia> # Multithreaded with @batch
       function axpy_batch!(y, a, x)
           Trixi.@threaded for i in eachindex(y,x)
               @inbounds y[i] = a * x[i] + y[i]
           end
       end
ERROR: LoadError: UndefVarError: `@batch` not defined in `Trixi`
Suggestion: check for spelling errors or missing imports.
Hint: a global variable of this name also exists in Polyester.
in expression starting at /home/lampert/.julia/dev/Trixi/src/auxiliary/auxiliary.jl:246
```
main branch:
```julia
julia> using Trixi

julia> # Multithreaded with @batch
       function axpy_batch!(y, a, x)
           Trixi.@threaded for i in eachindex(y,x)
               @inbounds y[i] = a * x[i] + y[i]
           end
       end
axpy_batch! (generic function with 1 method)
```
So, I go back to the original version and put `@batch` to the functions to be skipped by the test from ExplicitImports.jl.

Reply:
Done in [9a3ca08](https://github.com/trixi-framework/Trixi.jl/pull/1875/commits/9a3ca0866610fbec3868e1475a3ccaaa3fecaa51).